### ISSUE TITLE
Stack layout

### ISSUE THREAD

**Issue by mergesort** [zhxnlai/ZLSwipeableView#27]
Hi there, been using the library for a bit and really liking it.

I was wondering how the layout the views that are behind the top are determined? In this stack, sometimes they're all from the left, sometimes they're peeking in from the right. What I'd really like to mimic is the way it was in the demo, where sometimes they're peeking from the left, sometimes from the right, in an almost concentric fashion.
![layout](https://cloud.githubusercontent.com/assets/716513/6333886/d3cd5504-bb5e-11e4-82ae-db26e2a50a77.PNG)

Thanks so much for library!

**Comment by zhxnlai** [comment]
Thanks for reporting this! 

That is weird. I have never had this problem before. The subviews are default to be centered at its parent view (the ZLSwipeableView instance).  What is the frame of the ZLSwipeableView? I think it may be causing the problem.

**Comment by mergesort** [comment]
@zhxnlai The ZLSwipeableView's frame is `(20 10; 280 280)` instance is inside a UITableView headerView which has a frame of `(0 0; 320 300)`. I've tried it with both auto layout, and frame based layout, but end up with that issue.

**Comment by mergesort** [comment]
@zhxnlai I followed the example more closely and got it to work. I think the behavior is a bit strange.
Before in `nextViewForSwipeableView:` I was creating a square view, and returning it. I added a container view, and then added my square view to the container view, and returned  the container view in `nextViewForSwipeableView:`, and the layout worked. I'm not sure if it makes sense to require this container view for a proper layout, or that is a bug. Let me know if there's anything more I can do to help?
